apps/src/dev/index.html.ejs

```diff
@@ -10,7 +10,7 @@
   <li><a href="/studio?locale=en_us&dir=ltr&level=full_sandbox&skin=studio">Studio</a></li>
   <li><a href="/jigsaw?locale=en_us&dir=ltr&level=1&skin=jigsaw">Jigsaw</a></li>
   <li><a href="/calc?locale=en_us&dir=ltr&level=example1&skin=calc">Calc</a></li>
-  <li><a href="/applab?locale=en_us&dir=ltr&level=ec_simple&skin=applab">Applab</a></li>
-  <li><a href="/netsim?locale=en_us&dir=ltr&level=playground&skin=netsim">Internet Simulator (In Development)</a></li>
+  <li><a href="/applab?locale=en_us&dir=ltr&level=ec_simple&skin=applab"><s>Applab</s></a> <em>Not currently supported outside of dashboard; run bin/dashboard-server and access via http://localhost:3000/projects/applab/new</em></li>
```

Comment:
nit: let's send them to `/projects/applab` (without `/new`), since they probably don't need to create a new project every time.